blueprints/frost-component/files/__root__/__path__/__name__.js

```diff
@@ -1,13 +1,34 @@
 import Ember from 'ember'
-import FrostEvents from 'ember-frost-core/mixins/frost-events'
-import {
-  PropTypes
-} from 'ember-prop-types'
+const { Component } = Ember
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import layout from '<%= templatePath %>'
 
-const {
-  Component
-} = Ember
+const <%= classifiedModuleName %> = Component.extend(PropTypesMixin, {
+
+  // == Dependencies ==========================================================
+
+  // == Component properties ==================================================
+
+  layout,
+
+  // == State properties ======================================================
+
+  propTypes: {
+    hook: PropTypes.string
+  },
+
+  getDefaultProps () {
+    return {
+      hook: '<%= dasherizedModuleName %>'
+    }
+  }
+
+  // == Functions =============================================================
+
+  // == Events ================================================================
+
+  // == Actions ===============================================================
 
-<%= importTemplate %>
-export default Component.extend(FrostEvents, {<%= contents %>
 })
+
+export default <%= classifiedModuleName %>
```

Comment:
OK, I can see that, for single properties. I doubt I'd ever see that two positional properties would be better than named properties though. Unless, of course, it's something like `add` and it doesn't matter which comes first. Seems like you'd know that ahead of time though, right? Like at the time you generate the component? Could it be an option in the blueprint? I'd hate to see all those pointless variable declarations just for the edge case of when you want positional properties.